modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -121,6 +121,47 @@ struct GInferBase {
     }
 };
 
+struct InOutInfo
+{
+    std::vector<std::string> in_names;
+    std::vector<std::string> out_names;
+};
+
+class GInferInputs
+{
+public:
+    cv::GMat& operator[](const std::string& name) { return in_blobs[name]; }
```

Comment:
GMat is copiable, I'd return a copy safely.

Reply:
Fixed